pulsar-broker/src/test/java/org/apache/pulsar/client/impl/TopicsConsumerImplTest.java

```diff
@@ -163,6 +168,72 @@ public void testGetConsumersAndGetTopics() throws Exception {
         consumer.close();
     }
 
+    @Test
+    public void testMaxAcknowledgmentGroupSizeInvalid() {
+        try {
+            pulsarClient.newConsumer()
+                    .maxAcknowledgmentGroupSize(0);
+            fail("Should throw exception");
+        } catch (IllegalArgumentException e) {
+            // expect exception
+            assertEquals(e.getMessage(), "acknowledgementsGroupSize needs to be > 0");
+        }
+    }
```

Comment:
Let's move this test to `org.apache.pulsar.client.impl.ConsumerBuilderImplTest`

Reply:
OK, I will fix

Reply:
Fixed, PTAL,thanks! @RobertIndie